custom_components/elasticsearch/diagnostics.py

```diff
@@ -0,0 +1,22 @@
+"""Diagnostics for the Elasticsearch integration."""
+
+from homeassistant.components.diagnostics import async_redact_data
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_API_KEY,
+    CONF_PASSWORD,
+    CONF_URL,
```

Comment:
**question** Is there a need to redact the URL? The URL is currently a part of the config entry's title/display name, so this should already be "public"

Reply:
I was a little worried about having the URL and the fact that there is no authentication in the same diagnostic message but I'll take another look at this and remove the URL for now